src/mongo/db/storage/mmap_v1/mmap_windows.cpp

```diff
@@ -184,8 +184,8 @@ void* MemoryMappedFile::map(const char* filenameIn, unsigned long long& length)
     strncpy(filename, filenameIn, 255);
     filename[255] = 0;
     {
-        size_t len = strlen(filename);
-        for (size_t i = len - 1; i >= 0; i--) {
+        int len = strlen(filename);
```

Comment:
I'm sorry, if I don't understand you clearly. Will it be better, if I change only line 188?